themes/openy_themes/openy_carnation/src/scss/paragraphs/_branch_contact_info.scss

```diff
@@ -0,0 +1,200 @@
+.paragraph--type--branch-contacts-info {
```

Comment:
I can't find any css changes in the PR
I believe there is a step missing - need to recompile css

Reply:
@aleevas

Reply:
@podarok thank you for review.
Could you help me to understand that I need to do more?
Should I just make a recompile style and add that changes (css files) to my PR?

Reply:
@podarok
I've re-compiled styles.
Is it what I should did?
Please check it again.

Reply:
Yes, this is great now, thank you @aleevas